dvc/command/fsck.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+from dvc.command.common.base import CmdBase
+from dvc.data_cloud import file_md5
+
+
+class CmdFsck(CmdBase):
+    def run(self):
+        if not self.args.targets and not self.args.all:
```

Comment:
We should probably just check all files if no targets are specified. There is no need in a separate --all option, as that behavior fits nicely without explicit option.